data/campaigns/The_Hammer_of_Thursagan/scenarios/12_The_Underlevels.cfg

```diff
@@ -1036,151 +1113,269 @@
-            [primary_unit]
-                x,y=$x1,$y1
-            [/primary_unit]
-        [/fire_event]
+        [message]
+            speaker=unit
+            message= _ "My lord, the dirtgrubbers have entered our sanctuary!"
+        [/message]
 
-        [fire_event]
-            name=teleport_runes_use
-            [primary_unit]
-                x,y=$x1,$y1
-            [/primary_unit]
-        [/fire_event]
+        [message]
+            speaker=Karrag
+            message= _ "Excellent! Their blood will provide plenty of fuel for my ritual!"
+        [/message]
+
+        # Karrag's initial wave
+        [gold]
+            side=2
+            amount={ON_DIFFICULTY 100 125 150}
+        [/gold]
     [/event]
 
     [event]
         name=sighted
         [filter]
-            id=Aragoth
+            id=Karrag
         [/filter]
         [filter_second]
             side=1
         [/filter_second]
 
-        [gold]
-            side=4
-            amount=300
-        [/gold]
+        [message]
+            speaker=Karrag
+            message= _ "So, the usurpers have arrived at last."
+        [/message]
 
         [message]
-            speaker=Aragoth
-            message= _ "Our master’s ritual must not be interrupted. Stop them!"
+            speaker=Aiglondur
+            message= _ "We are not the usurpers but the liberators of Kal Kartha! We are here to free our kin from your dark sorcery!"
+        [/message]
+
+        [message]
+            speaker=Karrag
+            message= _ "You insolent fool! It is I who have led Kal Kartha to glory and I who have cleansed the lands of the true people from all unworthy scum! Kal Kartha owes me everything!"
+        [/message]
+
+        [message]
+            speaker=Angarthing
+            message= _ "No! You are the one who owes our brethren their freedom! You have ensnared their minds with your foul magic and now we are here to put an end to your tyrannical rule!"
         [/message]
+
+        [message]
+            speaker=Karrag
+            message= _ "Any enemy of my rule is no better than the filthiest of dirtgrubbers. You no longer belong to the true people. You will become my slaves!"
+        [/message]
+
+        [message]
+            speaker=Aiglondur
+            message= _ "Not if we stop you here. AXES UP!"
+        [/message]
+
+        # Give Karrag more gold
+        [gold]
+            side=2
+            amount={ON_DIFFICULTY 100 150 200}
+        [/gold]
     [/event]
 
     [event]
-        name=moveto
+        name=last breath
         [filter]
-            side=1
-            x=22,22
-            y=54,55
+            id=Karrag
         [/filter]
 
         [message]
-            speaker=unit
-            message= _ "Yon doors are big, tightly locked — and, I would imagine, well barricaded. This may take a while."
+            speaker=Karrag
+            message= _ "No! No! No! Dirtgrubbers must die! The true people must rule all!"
```

Comment:
Dirtgrubber is actually used in S11 without a po. Should I add one here or there or both?

Reply:
yes

Reply:
```suggestion
            # po: As used here, 'dirtgrubber' is a disparaging term used by ??? for ???.
```

Or something like that

Reply:
If I were being very rigorous, I'd even have a po comment about 'true people' being the name for themselves because they believe all others are inferior copies.

You have to ask yourself: to a non-native speaker could this word or term be misunderstood. For made-up terms such as dirtgrubber, even though it's a run-on of two common words, the answer is probably, 'yes'. And 'true people' here could be. "You may be a true people, but you are not the True People!"

Reply:
I added a po.